components/compositing/compositor.rs

```diff
@@ -1620,6 +1627,15 @@ impl IOCompositor {
                                 }
                                 self.touch_handler.remove_touch_sequence(sequence_id);
                             },
+                            TouchSequenceState::PendingContextMenu(point) => {
```

Comment:
Do we not need to handle this in the `TouchEventResult::DefaultPrevented` branch for touch up events? AFAIK contextmenu should be dispatched after down, so even if `up` has `preventDefault`, we should still dispatch the context menu event, unless `down` called `preventDefault`.